src/components/TextListWithHeading.js

```diff
@@ -11,9 +12,12 @@ export default class TextListGroupWithHeading extends React.Component {
     }
     let value = thing[property];
     if (!value || (_.isArray(value) && !value.length)) {
-      return <div />;
-    } else if (_.isArray(thing[property])) {
-      let items = thing[property].map(item => (
+      return null;
+    } else if (_.isArray(value)) {
+      if (property === "has_components" || property === "process_methods") {
```

Comment:
Can we add a TODO here and a github issue to follow up with a serverside fix?  This is a horrible hack and we should ack it =).

Reply:
you mean beautiful hack ;)

Reply:
yeah definitely agree this is hacky, but since we are in the midst of developing the new site with a changed api, i'm not sure if it makes sense to log an api issue for the api in this state. i had to test this frontend fix against `api.particpedia.xyz` because my local api is not returning the same results. the api repo has now diverged from `api.particpedia.xyz` quite a bit, and i'm not sure if this is an issue for the new build/site. in any case, i will add a comment to explain why we are doing this in the frontend repo.

Reply:
ok, filed an issue and added a comment for context. https://github.com/participedia/api/issues/464